pom.xml

```diff
@@ -94,7 +94,7 @@
     <test.args></test.args>
     <src.output>${basedir}/target</src.output>
     <imageio.ext.version>1.1.24</imageio.ext.version>
-    <jaiext.version>1.0.22</jaiext.version>
+    <jaiext.version>1.0-SNAPSHOT</jaiext.version>
```

Comment:
I guess that was for testing.
make sure to update it to proper version once the new jai-ext is released with jiffle update.

Reply:
Yep